sagemaker_model_governance/model_card_with_model_package.ipynb

```diff
@@ -0,0 +1,426 @@
+    "    (5.0, 1),\n",
+    "    (5.5, 1),\n",
+    ")\n",
+    "training_data = np.array(raw_data).astype(\"float32\")\n",
+    "labels = training_data[:, 1]\n",
+    "\n",
+    "# upload data to S3 bucket\n",
+    "buf = io.BytesIO()\n",
+    "smac.write_numpy_to_dense_tensor(buf, training_data, labels)\n",
+    "buf.seek(0)\n",
+    "boto3.resource(\"s3\").Bucket(bucket).Object(os.path.join(prefix, \"train\")).upload_fileobj(buf)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "53fdc706-1c26-4b7a-b7b6-77dc353d9e2c",
+   "metadata": {},
+   "source": [
+    "### 2. Create a training job\n",
+    "Train a binary classification model with the training data from the previous step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "722aff01-defd-4a9e-8b06-c0d7b320cc0d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s3_train_data = f\"s3://{bucket}/{prefix}/train\"\n",
+    "output_location = f\"s3://{bucket}/{prefix}/output\"\n",
+    "container = retrieve(\"linear-learner\", sagemaker_session.boto_session.region_name)\n",
+    "estimator = sagemaker.estimator.Estimator(\n",
+    "    container,\n",
+    "    role=role,\n",
+    "    instance_count=1,\n",
+    "    instance_type=\"ml.m4.xlarge\",\n",
+    "    output_path=output_location,\n",
+    "    sagemaker_session=sagemaker_session,\n",
+    ")\n",
+    "estimator.set_hyperparameters(feature_dim=2, mini_batch_size=10, predictor_type=\"binary_classifier\")\n",
+    "estimator.fit({\"train\": s3_train_data})\n",
+    "print(f\"Training job name: {estimator.latest_training_job.name}\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "eb31a7f3-3633-43f9-8220-a31cebdc14cc",
+   "metadata": {},
+   "source": [
+    "### 2. Create a model package"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "13385e3b-898f-4272-8c6a-5b030d016465",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a model package group\n",
+    "model_package_group_name = \"test-notebook-model-package-group\"\n",
+    "sagemaker_session.sagemaker_client.create_model_package_group(\n",
+    "    ModelPackageGroupName=model_package_group_name\n",
+    ")\n",
+    "\n",
+    "# describe training job to get model_data_url and image\n",
+    "training_job_name = estimator.latest_training_job.name\n",
+    "training_job = sagemaker_session.sagemaker_client.describe_training_job(\n",
+    "    TrainingJobName=training_job_name\n",
+    ")\n",
+    "\n",
+    "model_data_url = training_job[\"ModelArtifacts\"][\"S3ModelArtifacts\"]\n",
+    "image = training_job[\"AlgorithmSpecification\"][\"TrainingImage\"]\n",
+    "\n",
+    "# model package request input object\n",
+    "create_model_package_input_dict = {\n",
+    "    \"ModelPackageGroupName\": model_package_group_name,\n",
+    "    \"ModelPackageDescription\": \"Test model package registered for integ test\",\n",
```

Comment:
update the description to something relevant, like `Test model package to be registered in model card`